### ISSUE TITLE
analysis of prenatal development stage survey

### ISSUE THREAD

**Issue by lauraclarke** [ebi-ait/hca-ebi-wrangler-central#64]
# Analysis of prenatal development stage survey

A survey about how we record prenatal development stage in the HCA metadata standard has been circulated

https://docs.google.com/forms/d/1TIhGq2_cIpet6xpC7XzvMmiMezVanAlSCYwe87lQVCw/edit

The survey will be closed on June 24th

We need to analyse the results make a presentation for the metadata community call on May 29th

- Are there any development stage terms that we should use but aren't capturing?
- Which are the most valuable development stage terms?
- What recommendations can we make about what development stage measurements should be obligatory and what is optional

## Acceptance criteria
- [  ] Analysis of survey results completed
- [  ] Individual respondees whose input would be valuable for the metadata call send specific invites
- [  ] Slides produced for the call
- [  ] Tickets created for any metadata standards change which have emerged from the call
- [  ] Contributor guidance on what terms to use updated

**Comment by lauraclarke** [comment]
@gabsie @ESapenaVentura Anything else needed here?

**Comment by gabsie** [comment]
Hi, @lauraclarke  and @ESapenaVentura - I think after the analysis, it will be good to share with the people on the call our proposal/announcement for metadata changes. (if such, after the analysis)

Basically, I see our proposal as a basis for the discussion etc. So if we do the analysis on 24/25, then we need to come up with an internal discussion and a proposal for the meeting. 

Gabs

**Comment by lauraclarke** [comment]
Yeah, I was hoping the next time before we make changes to the schema which involve how we capture experimental or biomaterial fields that we will circulate the proposal to the whole community for some period before we make the change

**Comment by lauraclarke** [comment]
@ESapenaVentura @gabsie do we have a set of recommendations then we can close this ticket?
